torchax/test/test_context.py

```diff
@@ -76,20 +74,15 @@ def random_op():
 
     # Result always expected to be the same for a jitted function because seeds
     # are baked in
-    torch.testing.assert_close(
-        torchax.tensor.j2t(random_jit()._elem),
-        torchax.tensor.j2t(random_jit()._elem),
-        atol=0,
-        rtol=0)
+    torch.testing.assert_close(random_jit(), random_jit(), atol=0, rtol=0)
 
   def test_generator_seed(self):
     with xla_env:
       x = torch.randn(2, 3, generator=torch.Generator().manual_seed(0))
       y = torch.randn(2, 3, generator=torch.Generator().manual_seed(0))
 
-    # Values will be different, but still check device, layout, dtype, etc
-    torch.testing.assert_close(
-        torchax.tensor.j2t(x._elem), torchax.tensor.j2t(y._elem))
+      # Values will be different, but still check device, layout, dtype, etc
```

Comment:
nit: Values will be the same in this case?

Reply:
Right. The comment is wrong. Removed.